unit_tests/test_api.py

```
@@ -2,29 +2,17 @@
+import pandas as pd
+
 warnings.filterwarnings('ignore')
 pd.set_option('display.max_columns', None) # to ensure console display all columns
 pd.set_option('display.float_format', '{:0.3f}'.format)
 pd.set_option('display.max_row', 50)
-from pathlib import Path
-import joblib
-from copy import deepcopy
-
 
 import requests
```

Ruff (0.8.2)

Module level import not at top of file

(E402)